src/main/java/frc/robot/commands/CoralIntakeCommand.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import frc.robot.subsystems.CoralIntakeSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class CoralIntakeCommand extends Command {
+  /** Creates a new CorralIntakeCommand. */
+  private final CoralIntakeSubsystem coralIntake;
+  private final double speed;
+  public CoralIntakeCommand(CoralIntakeSubsystem subsystem, double speed) {
```

Comment:
Speed shouldn't be a Command parameter. Just in the subsystem and make it a set number for the setSpeed method